### DISCUSSION TITLE
Bad variable name for traceback object

### DISCUSSION THREAD

**Discussion by 100backslash001** [pallets/flask#5323] [Ideas]
It seems that the name `tb` for the variable is not a very good name
```python
tb = sys.exc_info()[2]
```
I know it might be obvious to some that `sys.exc_info()` returns a traceback object as the third element, but maybe should change the name to something more obvious? For example `traceback_object`? This way, the reader of the code will understand more clearly what the `tb` variable is and will not have to waste time trying to understand what is stored in it.

**Comment by ThiefMaster** [comment]
No, `tb` is a pretty standard name for tracebacks in Python code. Also, if you do not know what it returns, you should read the docs and you'll know what it is.
